src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,98 @@
+package duke;
+
+import duke.core.Core;
+import duke.gui.DialogBox;
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+/**
+ * <h1>Duke Task Management Application</h1>
+ * Duke is an applications that allow the user to manage their task from the command line.
+ *
+ * @author Stanley Neoh Jia Jun
+ */
+public class Duke extends Application {
+    private Core core;
+    private ScrollPane scrollPane;
+    private VBox dialogContainer;
+    private TextField userInput;
+    private Button sendButton;
+    private Scene scene;
+    private Image user = new Image(this.getClass().getResourceAsStream("/images/niko.png"));
+    private Image duke = new Image(this.getClass().getResourceAsStream("/images/giga.png"));
+
+    private void userSays(String message) {
+        Label userText = new Label(message);
+        dialogContainer.getChildren().add(DialogBox.getUserDialog(userText, new ImageView(user)));
+    }
+
+    private void dukeSays(String message) {
+        Label dukeText = new Label(message);
+        dialogContainer.getChildren().add(DialogBox.getDukeDialog(dukeText, new ImageView(duke)));
+    }
+
+    private void handleUserInput() {
+        String cmd = userInput.getText();
+        String resp = core.respond(cmd);
+        userSays(cmd);
+        dukeSays(resp);
+        userInput.clear();
+    }
+
+    @Override
+    public void start(Stage stage) {
+        //Initialise
+        scrollPane = new ScrollPane();
```

Comment:
Maybe having separate methods for initialise, styling etc. would make the code easier to read?